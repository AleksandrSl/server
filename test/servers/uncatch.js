#!/usr/bin/env node

var Server = require('../../server')
new Server({
  env: 'test',
  uniqName: 'server',
  subprotocol: [1, 0],
  supports: [1]
})

new Promise(function (resolve, reject) {
  setTimeout(function () {
    reject(new Error('Test Error'))
  }, 10)
})
