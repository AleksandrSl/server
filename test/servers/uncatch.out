
 ERROR  1970-01-01 00:00:00 Test Error
        at Timeout._onTimeout (test/servers/uncatch.js:13:12)
        at ontimeout (timers.js:365:14)
        at tryOnTimeout (timers.js:237:5)
        at Timer.listOnTimeout (timers.js:207:5)

 INFO  1970-01-01 00:00:00 Shutting down Logux server
